packages/bundle/src/__tests__/addMetaTag/minimal.spec.js

```diff
@@ -0,0 +1,43 @@
+/** @jest-environment @happy-dom/jest-environment */
+
+describe('importing minimal bundle', () => {
+  beforeEach(() => {
+    jest.requireActual('../../../dist/botframework-webchat.minimal.js');
+  });
+
+  test('should have injected <meta> tag', () => {
+    const metaMap = new Map(
+      Array.from(document.querySelectorAll('meta'))
+        .filter(({ name }) => name.startsWith('botframework-webchat'))
+        .map(({ content, name }) => [name, content])
+    );
+
+    expect(metaMap.has('botframework-webchat:api')).toBe(true);
+    expect(metaMap.has('botframework-webchat:bundle')).toBe(true);
+    expect(metaMap.has('botframework-webchat:component')).toBe(true);
+    expect(metaMap.has('botframework-webchat:core')).toBe(true);
+
+    expect(
+      metaMap
+        .get('botframework-webchat:api')
+        .split(';')
+        .map(value => value.trim())
+    ).toEqual(expect.arrayContaining(['build-tool=tsup', 'module-format=esmodules']));
+
+    expect(
+      metaMap
+        .get('botframework-webchat:bundle')
+        .split(';')
+        .map(value => value.trim())
+    ).toEqual(expect.arrayContaining(['build-tool=tsup', 'module-format=esmodules']));
```

Comment:
Can we make it so for bundle we set this properly on the import target?
`../../../dist/botframework-webchat.minimal.js` should be clearly `commonjs`.

Reply:
Yap, fixed. We actually mark everything as `esmodules` and nothing on `commonjs`.